third_party/terraform/tests/resource_container_cluster_test.go.erb

```diff
@@ -2290,6 +2340,44 @@ resource "google_container_cluster" "primary" {
 `, projectID, clusterName)
 }
 
+<% unless version == 'ga' -%>
+func testAccContainerCluster_withNotificationConfig(clusterName string, topic string) string {
+	return fmt.Sprintf(`
+
+resource "google_pubsub_topic" "%s" {
+  name = "%s"
+}
+
+resource "google_container_cluster" "notification_config" {
+  name               = "%s"
+  location           = "us-central1-a"
+  initial_node_count = 3
+  notification_config {
+	pubsub {
+	  enabled = true
+	  topic = "google_pubsub_topic.%s.id"  
```

Comment:
Sorry, I missed this!  This shouldn't be in quotes, we want to extrapolate the value from `google_pubsub_topic.%s.id`, we don't want that string here.  Then it should be good!

Reply:
No worries! I removed the quotes in 3862c14